source/wifi/wifi_hal.c

```diff
@@ -104,6 +104,7 @@
 #define BW_FNAME "/nvram/bw_file.txt"
 
 #define PS_MAX_TID 16
+BOOL G_BS_ENABLE = FALSE ;
```

Comment:
Please create new PR for this change.
This is not related to wifi_startNeighborScan

Reply:
removed the BS related changes in the current PR